packages/light-client/src/getState.ts

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+import {TreeBacked} from "@chainsafe/ssz";
+import {phase0} from "@chainsafe/lodestar-types";
+import {deserializeProof} from "@chainsafe/persistent-merkle-tree";
+import {config} from "@chainsafe/lodestar-config/mainnet";
+
+const BEACON_URL = "http://localhost:9596";
```

Comment:
probably make this a configuration variable?

Reply:
This should be a class, `LightclientApiClient` that accepts the URL as constructor param